pkg/azureutils/azure_disk_utils.go

```diff
@@ -1386,3 +1386,8 @@ func GetDefaultDiskMBPSReadWrite(requestGiB int) int {
 	}
 	return bandwidth
 }
+
+func GenerateUniqueRecoverAnnotation() string {
+	uuid := uuid.NewUUID().String()
+	return consts.RecoverAnnotation + uuid
```

Comment:
Instead of generating a unique `recoverAnnotation` key by concatenating the annotation prefix with an uuid, we should use `recoverAnnotation` as the key and generated uuid as the value.

Reply:
Yes. Done.